contracts/src/token/erc721/extensions/consecutive.rs

```diff
@@ -385,9 +387,9 @@ impl Erc721Consecutive {
         Ok(previous_owner)
     }
 
-    /// Returns the next token_id to mint using [`Self::_mint_consecutive`]. It
```

Comment:
Warning: https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown

The function does not work with any param named `token_id`, so this should be treated as plain text